frappe/www/update-password.html

```diff
@@ -161,11 +161,10 @@ <h4 class="reset-password-heading">{{ _("Reset Password") if frappe.db.get_defau
 						.text("{{ _('Invalid Password') }}");
 				},
 				200: function(r) {
-					if (r.message && r.message.score) {
```

Comment:
Removed score condition. If score is 0 (which means bad password) it doesn't show indicator :woozy_face: 


Score is always present in response from what I can tell.